services/minespace-web/src/components/common/DocumentTable.js

```diff
@@ -34,12 +37,12 @@ export const DocumentTable = (props) => {
     },
     {
       title: "Category",
-      dataIndex: "variance_document_category_code",
+      dataIndex: props.categoryDataIndex,
       render: (text) => <div title="Category">{props.documentCategoryOptionsHash[text]}</div>,
     },
```

Comment:
Do you have to generalize this as well? "props.documentCategoryOptionsHash" assuming variances would have different document options than project summaries.

Reply:
As it stands whatever hash you pass in as props.documentCategoryOptionsHash will work here, I didn't rename that prop, but I can if it stands to be problematic.

Reply:
Ohhh that's passed in? Okay that works.